tools/localazy/config.json

```diff
@@ -128,17 +128,20 @@
     {
       "name" : ":features:messages:impl",
       "includeRegex" : [
-        "room_timeline_.*",
-        "screen_room_.*",
-        "screen\\.room\\..*",
-        "screen_dm_details_.*",
         "emoji_picker_category_.*",
-        ".*report_content_.*"
-      ],
-      "excludeRegex" : [
-        "screen_room_details_.*",
-        "screen_room_member.*",
-        "screen_dm_.*"
+        "report_content_.*",
+        "room_timeline_.*",
+        "screen_report_content_.*",
+        "screen_room_attachment.*",
+        "screen_room_encrypted.*",
+        "screen_room_invite.*",
+        "screen\\.room\\.mention.*",
+        "screen_room_message.*",
+        "screen_room_no_permission.*",
+        "screen_room_reactions.*",
+        "screen_room_retry.*",
+        "screen_room_timeline.*",
+        "screen_room_typing.*"
```

Comment:
Could be nice to remove some strings key on Localazy to have a common prefix like `screen_room_timeline_`, but this require to do some change in the codebase both on Android and iOS codebase.

Reply:
(will do in https://github.com/element-hq/element-x-android/issues/2535)